src/Sylius/Bundle/AdminApiBundle/Resources/config/routing/main.yml

```diff
@@ -67,6 +67,10 @@ sylius_api_product_variant:
     resource: "@SyliusAdminApiBundle/Resources/config/routing/product_variant.yml"
     prefix: /products/{productCode}
 
+sylius_api_product_review:
```

Comment:
I guess that `sylius_api_product_review` should be before `sylius_api_product_taxon_position` to keep alphabetical order